routes/services/habit/updateHabit.ts

```
@@ -1,6 +1,54 @@
+      return;
+    }
+
+    // 상태 값 검증
+    if (status !== undefined && !["DONE", "UNDONE"].includes(status)) {
+      res.status(400).send("유효하지 않은 상태값입니다.");
+      return;
+    }
+
+    const upData: { name?: String; status?: "DONE" | "UNDONE" } = {};
+    if (name !== undefined) upData.name = name;
+    if (status !== undefined) upData.status = status;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)